src/compiler/AST/Typed.elm

```diff
@@ -25,16 +30,20 @@ import Transform
 
 
 type alias ProjectFields =
-    { modules : Modules Expr }
+    { modules : Modules LocatedExpr }
 
 
-{-| Differs from Canonical.Expr by:
+{-| Differs from Canonical.LocatedExpr by:
 
-  - being a tuple of the underlying Expr\_ type and its type
+  - being a tuple of the underlying LocatedExpr\_ type and its type
```

Comment:
This seems like a search-and-replace gone too far. I'd rather move this comment under `Expr_` and make it talk about Expr_ instead of LocatedExpr.

Reply:
Nice catch !